common-app-lib/src/cmr/common_app/cache/cubby_cache.clj

```diff
@@ -1,7 +1,11 @@
 (ns cmr.common-app.cache.cubby-cache
   "An implementation of the CMR cache protocol on top of the cubby application. Cubby only supports
   persistence and returning strings so this automatically serializes and deserializes the keys and
-  values with EDN. Any key or value serializable to EDN is supported."
+  values with EDN. Any key or value serializable to EDN is supported. Operationally cubby uses
```

Comment:
Excellent. Good find, and this feels like a good engineering solution to the old problem/convention.